src/validator/MatchValidator.cpp

```diff
@@ -175,7 +179,7 @@ Status MatchValidator::buildEdgeInfo(const MatchPath *path,
         }
         Expression *filter = nullptr;
         if (props != nullptr) {
-            auto result = makeSubFilter(*alias, props);
+            auto result = makeSubFilter(*alias, props, "");
```

Comment:
Need not to update it here if you set the default parameter for `makeSubFilter` function